master/usage/configuration/index.md

```diff
@@ -1,11 +1,7 @@
 ---
-title: Configuring Calico
+title: Configuring Systems for use with Calico
```

Comment:
@caseydavenport Yes, I think it would be good to move the OpenStack content to usage/openstack/configuration.md.  Note there's also a remaining para about conntrack state, which I believe is not OpenStack-specific.